modules/opensearch/src/OpenSearchTingObject.php

```diff
@@ -133,7 +275,30 @@ public function getFormat() {
    * {@inheritdoc}
    */
   public function isPartOf() {
-    return $this->getRecordEntry('dcterms:isPartOf');
+    $search_only_fields = ['oss:sort'];
```

Comment:
This is used multiple times. Consider extracting this to a class variable or constant.